mk/update-travis-yml.py

```diff
@@ -121,6 +121,11 @@ def format_entries():
             %(sources)s"""
 
 def format_entry(os, target, compiler, rust, mode):
+    if os == "linux" and compiler == "gcc-5" and rust == "nightly" and mode == "DEBUG":
```

Comment:
Please write this as `kcov = os == "linux" and compiler == "gcc-5" ...` (i.e. without the `if...else`).

Why only Nightly? If there's a technical reason, please explain it in a comment. I guess I would prefer to (also) do it on stable, since stable is what people will be using (in the long term).

Similarly, please write a comment explaining why you picked gcc-5 and debug mode.

RE debug mode: Is the issue that kcov requires debug symbols to work, and we don't have debug symbols in release builds? If so, perhaps we should just change, as we should have debug symbols for release builds anyway.